<launch>

 <!-- joy node -->
  <node respawn="true"
		pkg="joy"
        type="joy_node"
        name="rover5_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="2" type="double"/>
  <param name="scale_angular" value="2" type="double"/>

<!-- server node -->
  <node pkg="Rover5_ROS"
		type="rover5_coms"
		name="rover5_coms"
		output="screen" >
  </node>

<!-- odometry node -->
  <node pkg="Rover5_ROS"
		type="rover5_odom"
		name="rover5_odom"
		output="screen">
  </node>

<!-- imu node -->
  <node pkg="Rover5_ROS"
		type="rover5_imu"
		name="rover5_imu"
		output="screen">
  </node>

<!-- sonar node -->
  <node pkg="Rover5_ROS"
		type="rover5_sonar"
		name="rover5_sonar"
		output="screen">
  </node>

<param name="robot_description" command="cat $(find Rover5_ROS)/urdf/rover5.urdf" />

<!-- RVIS -->
  <node pkg="rviz"
		type="rviz"
		name="modelvisualisation"
		output="screen" >
  </node>

</launch>
